src/types/build-output.ts

```diff
@@ -0,0 +1,7 @@
+export interface IBuildOutput {
```

Comment:
Please use `export default` for all of the errors.

Reply:
Oops! sorry dude your message is clear but it seems you mention interface types 🤔
now I'm not sure to change export interface to export default or not